creator-cards/ahmed-shah.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Ahmed Shah",
+  "sectionTitle": null,
+  "countryCode": "pk",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": "",
+  "tags": ".NET,C#,Azure",
+  "socials": {
+    "youtube": "",
+    "linkedin": "https://www.linkedin.com/in/ahmedshahjr/",
+    "twitter": "https://twitter.com/ahmedshahjr",
+    "github": "https://github.com/ahmedshahjr",
+    "mastodon": "",
+    "sessionize": ""
+  },
+  "channels": [
+    {
+      "name": "DevTo",
+      "url": "https://dev.to/ahmedshahjr"
+    }
+  ],
+  "feeds": [
+    {
+      "type": "LinkTagRssType",
```

Comment:
Great blog! Please choose dotnet as feed type here. You may also want to switch to the new format: https://github.com/WeAreDotnet/dotnet-creator-cards/blob/main/creator-cards/_creatorTemplate.json